cg/server/app.py

```diff
@@ -42,24 +40,31 @@ def configure_extensions(app: Flask):
 
 
 def register_blueprints(app: Flask):
+
+    if not app.config['CG_ENABLE_ADMIN']:
+        return
```

Comment:
What does this do?

Reply:
Aha! Ok, I think I get it :)

We exit here and set up auth+admin_views when we want to have admin views.